tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml

```diff
@@ -0,0 +1,13 @@
+---
+addr: '0x05d379b48a8622ae4018309e116420db4a38a225'
+decimals: 18
+description: New card and payment system with 1 mil total supply.
+links:
+- Email: mailto:info@elyte.biz
+- Facebook: https://www.facebook.com/EthereumLyte
+- Telegram: https://t.me/joinchat/E6tXj0dThqzzjVlWJRuxLw
+- Website: https://www.elyte.biz
+- Whitepaper: https://main.elyte.biz/?page_id=9
+- BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851
```

Comment:
Looks like you updated capitalization here, but our format is case-sensitive and prefers `Bitcointalk` instead:

```
Checking tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml
  a links entry should be one of a well-known type (WARN: can 'BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851' be better described by one of well-known types: frozenset({'Telegram', 'Reddit', 'Blog', 'Whitepaper', 'Twitter', 'Bitcointalk', 'WeChat', 'Facebook', 'Github', 'Website', 'Email', 'Linkedin', 'CoinMarketCap', 'Slack'})?)
  Complete.
1 targets x 19 tests = 49 runs, of which: 0 failures, 1 warnings, 0 skipped.
----------------------------------------------------------------------------------------
tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml:
  1) Non-critical issue: can 'BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851' be better described by one of well-known types: frozenset({'Telegram', 'Reddit', 'Blog', 'Whitepaper', 'Twitter', 'Bitcointalk', 'WeChat', 'Facebook', 'Github', 'Website', 'Email', 'Linkedin', 'CoinMarketCap', 'Slack'})?
        in test_link_key_known: a links entry should be one of a well-known type
----------------------------------------------------------------------------------------
```